diagrams/__init__.py

```diff
@@ -392,4 +394,111 @@ def _load_icon(self):
+        self.color = color
+        self.style = style
+
+    def __sub__(self, other: Union["Node", "Edge", List["Node"]]):
+        """Implement Self - Node or Edge and Self - [Nodes]"""
+        return self.connect(other)
+
+    def __rsub__(self, other: Union[List["Node"], List["Edge"]]) -> List["Edge"]:
+        """Called for [Nodes] or [Edges] - Self because list don't have __sub__ operators."""
+        return self.append(other)
+
+    def __rshift__(self, other: Union["Node", "Edge", List["Node"]]):
+        """Implements Self >> Node or Edge and Self >> [Nodes]."""
+        self.forward = True
+        return self.connect(other)
+
+    def __lshift__(self, other: Union["Node", "Edge", List["Node"]]):
+        """Implements Self << Node or Edge and Self << [Nodes]."""
+        self.reverse = True
+        return self.connect(other)
+
+    def __rrshift__(self, other: Union[List["Node"], List["Edge"]]) -> List["Edge"]:
+        """Called for [Nodes] or [Edges] >> Self because list of Edges don't have __rshift__ operators."""
+        return self.append(other, forward=True)
+
+    def __rlshift__(self, other: Union[List["Node"], List["Edge"]]) -> List["Edge"]:
+        """Called for [Nodes] or [Edges] << Self because list of Edges don't have __lshift__ operators."""
+        return self.append(other, reverse=True)
+
+    def append(self, other: Union[List["Node"], List["Edge"]], forward=None, reverse=None) -> List["Edge"]:
+        result = []
+        for o in other:
+            if isinstance(o, Edge):
+                o.color = self.color
+                o.label = self.label
+                o.style = self.style
+                o.forward = forward if forward is not None else o.forward
+                o.reverse = forward if forward is not None else o.reverse
+                result.append(o)
+            else:
+                result.append(Edge(o, forward=forward, reverse=reverse,
+                                   color=self.color, label=self.label, style=self.style))
+        return result
+
+    def connect(self, other: Union["Node", "Edge", List["Node"]]):
+        if isinstance(other, list):
+            for node in other:
+                self.node.connect(node, self)
+            return other
+        elif isinstance(other, Edge):
+            self.label = other.label
+            self.color = other.color
+            self.style = other.style
+            return self
+        else:
+            if self.node is not None:
+                return self.node.connect(other, self)
+            else:
+                self.node = other
+                return self
+
+    def to_dot(self) -> Dict:
+        dot = {}
+        if self.forward and self.reverse:
+            dot['dir'] = 'both'
+        elif self.forward:
+            dot['dir'] = 'forward'
+        elif self.reverse:
+            dot['dir'] = 'back'
+        else:
+            dot['dir'] = 'none'
+        if self.label:
+            dot['label'] = self.label
```

Comment:
The Graphviz shows me `Warning: Orthogonal edges do not currently handle edge labels. Try using xlabels.` message when using `label` with edge.

You should use `xlabel` instead.

Reply:
Done